tsdoc/schemas/tsdoc.schema.json

```diff
@@ -50,8 +50,7 @@
         "@[a-zA-Z][a-zA-Z0-9]*$": {
           "type": "boolean"
         }
-      },
-      "additionalItems": false
```

Comment:
Removing this?

Reply:
@iclanton after upgrading AJV, the validator now uses `strict` mode by default, which reports an error for this field because it has no effect.